### ISSUE TITLE
Adding Project Section.

### ISSUE THREAD

**Issue by ira-mane** [sumn2u/learn-javascript#63]
I am a learning developer and when it comes to learning a programming language I feel along with strong grip of theory and rules one should have hands on learning of that language.

As there is already a exercise section ,I feel there can be a project section where few small projects are suggested and explained so that the reader understand how they can use the learned concepts in application.

Please let me know your thoughts on this !!
Thankyou.

**Comment by somuyakhandelwal** [comment]
Collaborative Learning Projects

Hello @sumn2u,

I hope you're doing well. I'm excited about contributing to our community through meaningful projects. I believe in the power of collaboration and knowledge sharing.

I've come up with innovative project ideas that can greatly enhance the learning experience. These projects empower learners and create valuable resources.

Best regards,
somuya khandelwal

**Comment by sumn2u** [comment]
@ira-mane Thank you for taking the initiative. I appreciate the idea and see the benefits from a learner's perspective. Can you lead this chapter in co-ordination with @somuyakhandelwal?

**Comment by ira-mane** [comment]
Hello @sumn2u  Glad you liked the idea ,I really wanted to work on this , can you assign me this ?

**Comment by somuyakhandelwal** [comment]
yaa sure I would love to assist @ira-mane can you assign us on this

**Comment by somuyakhandelwal** [comment]
@sumn2u can you assign me too..

**Comment by sumn2u** [comment]
> @sumn2u can you assign me too..

If it's okay, I can divide this issue into two parts where each of you can work separately. Is that okay with both of you?

cc/ @somuyakhandelwal  @ira-mane

**Comment by ira-mane** [comment]
> > @sumn2u can you assign me too..
> 
> If it's okay, I can divide this issue into two parts where each of you can work separately. Is that okay with both of you?
> 
> cc/ @somuyakhandelwal @ira-mane

Yeah sure I don't have any problem. but exactly how would it be done ?

**Comment by somuyakhandelwal** [comment]
Yaa sure even I don't have any problem

**Comment by sumn2u** [comment]
> Yeah sure I don't have any problem. but exactly how would it be done ?

That's a great question! I was considering suggesting 10 projects (although this number is flexible) and proposing that @ira-mane complete the first 5 projects, while @somuyakhandelwal takes on the remaining 5. 

This will allow for separate pull requests, and we can always rebase to the accepted PR and have one's work on top of others.

**Comment by sumn2u** [comment]
@somuyakhandelwal The second part is [here](https://github.com/sumn2u/learn-javascript/issues/66).

**Comment by ira-mane** [comment]
I will work on this project 1) Random text Generator
2) Colour flipper
3) counter
4)Button functionality  

Is it okay @sumn2u  ??

**Comment by ira-mane** [comment]
Hellooo @somuyakhandelwal I will rebase on your accepted pr . 
Thankyou !!

**Comment by sumn2u** [comment]
> I will work on this project 1) Random text Generator 2) Colour flipper 3) counter 4)Button functionality
> 
> Is it okay @sumn2u ??

Wow, these three projects are interesting. You can go ahead.

**Comment by somuyakhandelwal** [comment]
> Hellooo @somuyakhandelwal I will rebase on your accepted pr . Thankyou !! Have you set any deadline for yourself on this ??

Hi @ira-mane 
Okay sure ,I will be adding all the projects by Monday morning

**Comment by ira-mane** [comment]
> > Hellooo @somuyakhandelwal I will rebase on your accepted pr . Thankyou !! Have you set any deadline for yourself on this ??
> 
> Hi @ira-mane Okay sure ,I will be adding all the projects by Monday morning What about you when you will be completing all the projects...?

I will complete it by tuesday.

**Comment by Kedarnath-Rothe** [comment]
Please assign to me

**Comment by barkhaaroraa** [comment]
hi is this issue still active? its been 2 weeks but no pr has been made. so if possible i'd like to work on this since it seems to be inactive at this point

**Comment by sumn2u** [comment]
@Bhumika-00 Are you interested in this?

**Comment by Bhumika-00** [comment]
Yes, I'm interested.

**Comment by raj-adi00** [comment]
I am interested and want to add some good projects here. Will you please assign me

**Comment by Bhumika-00** [comment]
I accept the assignment.

**Comment by raj-adi00** [comment]
@sumn2u  Thank you for assigning me. I want to know what do you expect in frontend. Currently i have an idea that i will make a json file because you are not using database. Each project will have key for html css and js code. If you want something else Please tell i will update accordingly

**Comment by SuryaNarayananDev** [comment]
can i add project section in that project ?
